pkg/apply/apply.go

```diff
@@ -72,6 +72,9 @@ type NewChangesTracker struct {
 	NumPartitions     int               `json:"numPartitions"`
 	ReplicationFactor int               `json:"replicationFactor"`
 	ConfigEntries     *[]NewConfigEntry `json:"configEntries"`
+	// Error stores if an error occurred during topic update
+	Error bool          `json:"error"`
+	ErrorMessage string `json:"errorMessage"`
```

Comment:
to support error notifications from empty changes, `NewChangesTracker` stores if an error occurred along with an error message.
This is filled in if an error is caught.